report_xlsx/report/report_partner_xlsx.py

```diff
@@ -10,7 +10,7 @@ class PartnerXlsx(models.AbstractModel):
     _description = "Partner XLSX Report"
 
     def generate_xlsx_report(self, workbook, data, partners):
-        sheet = workbook.add_worksheet('Report')
+        sheet = workbook.add_worksheet("Report")
         for i, obj in enumerate(partners):
-            bold = workbook.add_format({'bold': True})
+            bold = workbook.add_format({"bold": True})
```

Comment:
These changes should be part of the previous commit `[12.0][FIX] Report xlsx res_partner`

Reply:
Hi @sebalix , Thank you for review. I just fixed it.